contracts/exchange/test/assertion_wrappers/fill_order_wrapper.ts

```diff
@@ -0,0 +1,282 @@
+import { artifacts as proxyArtifacts , ERC1155ProxyWrapper, ERC20Wrapper, ERC721Wrapper } from '@0x/contracts-asset-proxy';
+import { artifacts as erc20Artifacts } from '@0x/contracts-erc20';
+import { artifacts as erc721Artifacts } from '@0x/contracts-erc721';
+import {
+    chaiSetup,
+    FillEventArgs,
+    LogDecoder,
+    OrderStatus,
+    orderUtils,
+    Web3ProviderEngine,
+} from '@0x/contracts-test-utils';
+import { orderHashUtils } from '@0x/order-utils';
+import { FillResults, SignedOrder } from '@0x/types';
+import { BigNumber } from '@0x/utils';
+import { Web3Wrapper } from '@0x/web3-wrapper';
+import * as chai from 'chai';
+import { LogWithDecodedArgs, TransactionReceiptWithDecodedLogs, ZeroExProvider } from 'ethereum-types';
+import * as _ from 'lodash';
+
+import { artifacts, ExchangeContract } from '../../src';
+import { BalanceStore } from '../balance_stores/balance_store';
+import { BlockchainBalanceStore } from '../balance_stores/blockchain_balance_store';
+import { LocalBalanceStore } from '../balance_stores/local_balance_store';
+
+chaiSetup.configure();
+const expect = chai.expect;
+
+export class FillOrderWrapper {
+    private readonly _exchange: ExchangeContract;
+    private readonly _blockchainBalanceStore: BlockchainBalanceStore;
+    private readonly _web3Wrapper: Web3Wrapper;
+    /**
```

Comment:
Nit: Should we start adding lines between functions? Personally I'd prefer it.

Reply:
Nice, I'm happy you mentioned this lol. I've been doing this for consistency but I'm more than happy to start adding newlines now. :)